Makefile

```diff
@@ -106,9 +106,11 @@ install_plugins: .pulumi/bin/pulumi
 lint_provider: provider
 	cd provider && golangci-lint run -c ../.golangci.yml
 
-provider: tfgen install_plugins
+provider_no_deps:
```

Comment:
I'd be excite to land this but this needs to go into ci-mgmt since otherwise the nightly job will revert this change.